unobot.py

```diff
@@ -109,7 +109,8 @@
                         "use color codes at all."],
     'PLAY_SYNTAX':     "Command syntax error. You must use e.g. %pplay r 3 or %pplay w y.",
 }  # yapf: disable
-COLORED_CARD_NUMS = ['1', '2', '3', '4', '5', '6', '7', '8', '9', 'R', 'S', 'D2']
+# don't sort card values to ensure 0 is ALWAYS first
+COLORED_CARD_NUMS = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'R', 'S', 'D2']
```

Comment:
0 belongs in the special cards by virtue of the fact that it appears less often than the other numbers, but I'd need to review the rest of the code to see if there would be any unintended consequences of putting it there (besides eliminating the list slice and concatenation below).